browser/README.md

```diff
@@ -18,20 +15,51 @@ $ make release
 This will produce a file that looks like `keybase-chrome-1.2.3.zip` (except with
 whatever the current version is set in the `manifest.json`).
 
-Upload that file to the Chrome extension dashboard, hit publish, and we're done.
+Upload that file to the Chrome or Firefox extension dashboard, hit publish, and
+we're done.
+
 
+## Manual Install and QA
 
-## Manual Install
+Build a fresh extension to load into your browsers:
 
-### Chrome Extension
+
+```shell
+$ yarn; make release
```

Comment:
Can you confirm that this actually works? :D I forget if there's a bit more setup required upfront.

Reply:
works!